1.fee_bumping/fee_bumping.md

```diff
@@ -153,33 +153,224 @@ Bitcoin blockchain, such as exchanges or custodians:
+  of some of those transactions as additional inputs to bump the fee on its other
+  transactions.
 
-TODO
+> 3. The replacement transaction pays an absolute fee of at least the sum paid
+> by the original transactions.
+
+- implication 1: if outputs from the original transaction have been spent, then
+  the replacement transaction must exceed both the *feerate* and *total fee* of
+  the original transaction and its descendants. If the child transaction is large,
+  or there are many descendant transactions, then the total fee required to
+  replace the original transaction could be very large. This can make it
+  uneconomical to replace the original transaction and so is known as
+  *transaction pinning*.
+
+> 4. The replacement transaction must also pay for its own bandwidth at or
+> above the rate set by the node's minimum relay fee setting. For example,
+> if the minimum relay fee is 1 satoshi/byte and the replacement transaction
+> is 500 bytes total, then the replacement must pay a fee at least 500
+> satoshis higher than the sum of the originals.
+
+Rationale: this is an anti-DoS measure to prevent an adversary from repeatedly
+sending replacement transactions with miniscule feebumps and making the peer
+revalidate many versions of the same transaction.
+
+- implication 1: for transactions at low feerates, the proportional change
+  in feerate from bumping is large. A transaction with a feerate of
+  1 sat/byte needs to *double* its fee to satisfy this rule. For very
+  large transactions (eg large consolidations), then the amount of money
+  spent on fees could be high.
+
+  Although BIP 125 specifies that the transaction must increase the feerate
+  by the *minimum relay fee*, Bitcoin Core actually has a separate parameter
+  for the amount that the fee needs to be increased by, called the
+  *incrementalrelayfee*. By default this is set to the same value as the
+  *minimum relay fee* - 1 sat/byte.
+
+> 5. The number of original transactions to be replaced and their descendant
+> transactions which will be evicted from the mempool must not exceed a
+> total of 100 transactions.
+
+Rationale: this is an anti-DoS measure to prevent an adversary from sending
+replacement transactions that cause their peer to walk through many
+transactions in their mempool. This scan through the mempool needs to be done
+before calculating whether the replacement transaction has enough fee to
+replace the original transactions, so an adversary could cause the peer to walk
+the mempool without causing transactions to be replaced, which would be a free
+attack.
 
 ## User Experience Recommendations
 
 Even wallets and services that do not themselves support creating opt-in RBF or
 replacement transactions should present a clear and accurate experience to
 their users when dealing with RBF transactions:
 
-- wallets that receive transactions that have opt-in RBF signalled may
+- wallets that receive transactions that have opt-in RBF signaled may
```

Comment:
Do you mean "may display" in the sense that they might not display them at all, or not display a warning?

IMHO, it's very dangerous to display unconfirmed opt-in RBF transactions without a warning, and it would be much more preferable to not display them at all rather than without a warning. Against all recommendations, way too many users trust unconfirmed transactions fully.